pkgs/by-name/lu/lunarvim/package.nix

```diff
@@ -0,0 +1,101 @@
+{ fd
+, fetchFromGitHub
+, fzf
+, git
+, lib
+, makeWrapper
+, neovim
```

Comment:
Maybe this should be `neovim-unwrapped`? (I'm not sure about this)

Reply:
https://github.com/NixOS/nixpkgs/blob/ad69bc581b2e3c90dc397ddfc1df7369c19e2ba6/pkgs/top-level/all-packages.nix#L36390